code/__DEFINES/mobs.dm

```diff
@@ -403,8 +403,8 @@
 #define OFFSET_NECK "neck"
 
 //MINOR TWEAKS/MISC
-#define AGE_MIN 17 //youngest a character can be
```

Comment:
Will preferences automatically update 17 year old characters to 18?

Reply:
yes afaik

Reply:
please check so